README.md

```diff
@@ -14,66 +14,167 @@ It can be run inside a cluster (under a Service Account) or outside (via any pro
 ## Installation
 Kai can be run as a CLI, Docker Container, or Helm Chart
 
-By default, Kai will look for a Kubeconfig in the home directory to use to authenticate (when run as a CLI). 
+By default, Kai will look for a Kubeconfig in the home directory to use to authenticate (when run as a CLI).
```

Comment:
Guess we should standardize on KAI for the name of the project 👍

Reply:
Done!